### ISSUE TITLE
[web ui] Add label on homepage: Habitat is a fully open source project

### ISSUE THREAD

**Issue by apriofrost** [habitat-sh/habitat#2568]
![image](https://user-images.githubusercontent.com/9401459/26995252-70813c30-4d20-11e7-9bfd-973e5cfb4879.png)

Adding this label above the tagline will better inform new users about the OSS status for Habitat. 
Mockup link: https://chef.invisionapp.com/d/main#/console/9499228/229572977/preview

**Comment by ryankeairns** [comment]
Let's set this up as an A/B test so that we can measure the improvement in parallel to the current state. That will give us a clearer indication of its effect as opposed to running this sequentially where other factors can occur over the differing time periods.

**Comment by cnunciato** [comment]
It might also be nice to have this link off to a Wiki page explaining things around licensing, etc.

**Comment by apriofrost** [comment]
@cnunciato Yeah. We already have a licensing page that we can link to: https://www.habitat.sh/legal/licensing/

**Comment by eeyun** [comment]
Excellent. Thats information we should definitely duplicate in the github wiki too! I think linking within the site is probably good though for this wonderful header.

**Comment by cnunciato** [comment]
Is this one still relevant, given the recent release?

**Comment by ryankeairns** [comment]
@cnunciato let's close this out and see how the new site performs in this regard. We do have a github link top left, a start button, pricing page, and no longer have a 'try habitat' demo button which some thought made it feel like a paid only product.
